src/main/java/gift/service/WishService.java

```diff
@@ -0,0 +1,42 @@
+package gift.service;
+
+import gift.dto.WishRequest;
+import gift.exception.product.ProductNotFoundException;
+import gift.model.Product;
+import gift.model.Wish;
+import gift.repository.ProductDao;
+import gift.repository.WishDao;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+
+@Service
+public class WishService {
```

Comment:
```suggestion
import java.util.List;

@Service
public class WishService {
```

개행 하나도 의미전달의 의미가 있습니다. 이러한 부분들을 생각해보시면 좋을거 같아요~